Tasks/DownloadPackageV0/task.loc.json

```diff
@@ -63,7 +67,7 @@
       "endpointId": "tfs:feed",
       "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds",
       "resultSelector": "jsonpath:$.value[*]",
```

Comment:
I wonder if project scoped feeds can be excluded in the jsonpath query instead

Reply:
I tried using value[?(@project == null)], however it doesn't work. It works the other way around value[?(@project != null)], but not the way we want it to.